cookbooks/aws-parallelcluster-platform/files/dcv/pcluster_dcv_authenticator.py

```diff
@@ -361,13 +361,17 @@ def _is_session_valid(user, session_id):
         # TODO change this method if DCV updates his behaviour.
         """
         logger.info("Verifying NICE DCV session validity..")
+
+        # Query by uid rather than username to avoid truncation by ps command
+        uid = subprocess.check_output(["id", "-u", user]).decode("utf-8").strip()
```

Comment:
here you should use: `/usr/bin/id` as first argument, see https://bandit.readthedocs.io/en/1.7.5/plugins/b607_start_process_with_partial_path.html

Reply:
Could you also test by passing `shell=True` to `check_output`?
see https://bandit.readthedocs.io/en/1.7.5/plugins/b603_subprocess_without_shell_equals_true.html